File: Python/FizzBuzz.py
#!/usr/bin/python3

def fizzBuzz():
    print ("Welcome to Fizz Buzz!")

    fizzNum=int(input("What number to Fizz Buzz?: "))

    i = 0

    while (i <= fizzNum):
        if i % 15 == 0:
            print("Fizz Buzz")
            i = i + 1
        elif i % 3 == 0:
            print("Fizz")
            i = i + 1
        elif i % 5 == 0:
            print("Buzz")
            i = i + 1
        else:
            print(i)
            i = i + 1

fizzBuzz()

File: C++/FizzBuzz/fizzbuzz.cpp
#include <iostream>

using namespace std;

int main () 
{
    int maxNum = 100;

    cout << "Welcome to FizzBuzz! " << endl;

    for (int i = 0; i <= maxNum; i++)
    {
        if (i % 15 == 0)
        {
            cout << "Fizz Buzz" << endl;
        }
        else if (i % 3 == 0)
        {
            cout << "Fizz" << endl;
        }
        else if (i % 5 == 0)
        {
            cout << "Buzz " << endl;
        }
        else
        {
            cout << i << endl;
        }
    }

    return 0;
}
